src/main/java/com/amazon/jenkins/ec2fleet/CloudNames.java

```diff
@@ -18,11 +24,29 @@ public static String generateUnique(final String defaultName) {
             : Collections.emptySet();
     String uniqName = defaultName;
 
-    int suffix = 1;
     while (usedNames.contains(uniqName)) {
-      uniqName = defaultName + "-" + suffix++;
+      uniqName = defaultName + "-" + generateSuffix();
     }
 
     return uniqName;
   }
+
+  public static String processCloudName(String name) {
```

Comment:
We can combine this and generateUniqueCloud name but I left them separate in case we eventually need some other form of name processing. If we don't think this is necessary I will go ahead and combine them.

Reply:
Yea! I think it makes sense to just use `generateUnique` in the Cloud constructors. Left a suggestion.